error when updating to RN 0.77.0

```

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-photo-manipulator:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 35s
error Failed to install the app. Command failed with exit code 1: gradlew.bat app:installDebug -PreactNativeDevServerPort=8081 -PreactNativeArchitectures=x86_64     
No modules to process in combine-js-to-schema-cli. If this is unexpected, please check if you set up your NativeComponent correctly. See combine-js-to-schema.js for how codegen finds modules.
No modules to process in combine-js-to-schema-cli. If this is unexpected, please check if you set up your NativeComponent correctly. See combine-js-to-schema.js for how codegen finds modules.
No modules to process in combine-js-to-schema-cli. If this is unexpected, please check if you set up your NativeComponent correctly. See combine-js-to-schema.js for how codegen finds modules.
Note: Some input files use or override a deprecated API. Note: Recompile with -Xlint:deprecation for details. Note: D:\RN\gallery-comics\node_modules\@react-native-async-storage\async-storage\android\src\javaPackage\java\com\reactnativecommunity\asyncstorage\AsyncStoragePackage.java uses unchecked or unsafe operations. Note: Recompile with -Xlint:unchecked for details. Note: Some input files use or override a deprecated API. Note: Recompile with -Xlint:deprecation for details. D:\RN\gallery-comics\node_modules\@react-native-community\netinfo\android\src\main\java\com\reactnativecommunity\netinfo\NetInfoModule.java:47: warning: [removal] onCatalystInstanceDestroy() in NativeModule has been deprecated and marked for removal public void onCatalystInstanceDestroy() { ^ Note: Some input files use or override a deprecated API. Note: Recompile with -Xlint:deprecation for details. 1 warning Note: Some input files use or override a deprecated API. Note: Recompile with -Xlint:deprecation for details. Note: Some input files use or override a deprecated API. Note: Recompile with -Xlint:deprecation for details. Note: Some input files use or override a deprecated API. Note: Recompile with -Xlint:deprecation for details. Note: Some input files use or override a deprecated API. Note: Recompile with -Xlint:deprecation for details. Note: D:\RN\gallery-comics\node_modules\@react-native-firebase\crashlytics\android\src\main\java\io\invertase\firebase\crashlytics\ReactNativeFirebaseCrashlyticsModule.java uses or overrides a deprecated API. Note: Recompile with -Xlint:deprecation for details. Note: Some input files use or override a deprecated API. Note: Recompile with -Xlint:deprecation for details. Note: Some input files use or override a deprecated API. Note: Recompile with -Xlint:deprecation for details. Note: Some input files use unchecked or unsafe operations. Note: Recompile with -Xlint:unchecked for details. e: file:///D:/RN/gallery-comics/node_modules/react-native-photo-manipulator/android/src/main/java/com/guhungry/rnphotomanipulator/RNPhotoManipulatorModule.kt:167:39 Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type 'com.facebook.react.bridge.ReadableMap?'. e: file:///D:/RN/gallery-comics/node_modules/react-native-photo-manipulator/android/src/main/java/com/guhungry/rnphotomanipulator/RNPhotoManipulatorModule.kt:167:74 Only safe (?.) or non-null asserted (!!.) calls are allowed on a nullable receiver of type 'com.facebook.react.bridge.ReadableMap?'. e: file:///D:/RN/gallery-comics/node_modules/react-native-photo-manipulator/android/src/main/java/com/guhungry/rnphotomanipulator/RNPhotoManipulatorModule.kt:167:98 Argument type mismatch: actual type is 'com.facebook.react.bridge.ReadableMap?', but 'com.facebook.react.bridge.ReadableMap' was expected. Note: D:\RN\gallery-comics\node_modules\react-native-system-navigation-bar\android\src\main\java\com\reactnativesystemnavigationbar\SystemNavigationBarModule.java uses or overrides a deprecated API. Note: Recompile with -Xlint:deprecation for details. FAILURE: Build failed with an exception. * What went wrong: Execution failed for task ':react-native-photo-manipulator:compileDebugKotlin'. > A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction > Compilation error. See log for more details * Try: > Run with --stacktrace option to get the stack trace. > Run with --info or --debug option to get more log output. > Run with --scan to get full insights. > Get more help at https://help.gradle.org. BUILD FAILED in 35s.  
info Run CLI with --verbose flag for more details.
```

--- a/android/src/main/java/com/guhungry/rnphotomanipulator/RNPhotoManipulatorModule.kt
+++ b/android/src/main/java/com/guhungry/rnphotomanipulator/RNPhotoManipulatorModule.kt
@@ -163,7 +163,7 @@ class RNPhotoManipulatorModule(private val context: ReactApplicationContext) : R
             val output = bitmapFromUri(context, uri, mutableOptions())
 
             for (i in 0 until list.size()) {
-                val text = list.getMap(i)
+                val text = list.getMap(i) ?: continue
                 printLine(output, text.getString("text")!!, toPointF(text.getMap("position"))!!, text)
             }
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "react-native-photo-manipulator",
-  "version": "1.9.1",
+  "version": "1.9.2",
   "description": "React Native Image Processing API to edit photo programmatically for Android and iOS.",
   "source": "./src/index.ts",
   "main": "./lib/commonjs/index.js",

